spec/requests/admin/admin_portals_spec.rb

```diff
@@ -24,16 +24,12 @@
   end
 
   describe "Last deployed and Lastest Commit ID card" do
-    after do
+    before do
```

Comment:
As a side effect of removing the previous test I found that I've needed to set  `ForemInstance.instance_variable_set(:@deployed_at, nil)`. 

Honestly I'm not sure where the deployed at instance is bleeding in from the tests, I've tried every conceivable test run order and trick to see why this would be an issue. When I get into a debugger it even shows deployed_at as nil.

In the essence of not spending any more time on this, I've now just set it to nil prior to the test. I don't see any harm in setting to nil before and after the test to avoid it bleeding from and to anywhere. The same seems to be done in the ForemInstance spec. I can look into optimising this in more detail during cooldown.